common/djangoapps/student/models.py

```diff
@@ -1374,7 +1375,9 @@ def refundable(self):
         if GeneratedCertificate.certificate_for_student(self.user, self.course_id) is not None:
             return False
 
-        #TODO - When Course administrators to define a refund period for paid courses then refundable will be supported. # pylint: disable=fixme
+        # If it is after the refundable window they should not be refunded.
+        if self._refund_window_end_date() and datetime.now() > self._refund_window_end_date():
```

Comment:
Don't call `self._refund_window_end_date` twice -- it's got multiple DB calls and a network call.